src/makieplotting.jl

```diff
@@ -307,6 +307,105 @@ function Makie.plot!(AR::Arrows{<:Tuple{<:MakiePlotter{dim}}}) where dim
     Makie.arrows!(AR, ps, ns, arrowsize=AR[:arrowsize], normalize=AR[:normalize], colormap=AR[:colormap], color=lengths, lengthscale=AR[:lengthscale])
 end
 
+"""
+    elementinfo(ip::Interpolation; kwargs...)
+    elementinfo(cell::AbstractCell; kwargs...)
+    elementinfo(ip::Type{Interpolation}; kwargs...)
+    elementinfo(cell::Type{AbstractCell}; kwargs...)
+
+- `plotnodes=true` controls if nodes of element are plotted
+- `strokewidth=2` strokwidth of faces/edges
+- `color=theme(scene, :linecolor)`
+- `markersize=30` size of the nodes
+- `textsize=60` textsize of node-, edges- and facelabels
+- `nodelabels=true` switch that controls plotting of nodelabels
+- `nodelabelcolor=:darkred`
+- `nodelabeloffset=(0.0,0.0)` offset of the nodelabel text relative to its associated node
+- `facelabels=true` switch that controls plotting of facelabels
+- `facelabelcolor=:darkgreen`
+- `facelabeloffset=(0,0)` offset of the facelabel text relative to its associated face middlepoint
```

Comment:
Maybe the offset for the face label and node label should have different defaults, so they do not overlap by default.

Reply:
the default position never coincide so they should never overlap by default even thought the offset value is the same

Reply:
```julia
julia> using Ferrite, FerriteViz, GLMakie

julia> elementinfo(Lagrange{3,RefCube,2}())
```

results in

![Screenshot from 2022-10-17 21-17-27](https://user-images.githubusercontent.com/9196588/196263785-c58423c6-adfa-4b17-b5a7-9971fd65c01b.png)